masterlist.yaml

```diff
@@ -28,10 +28,84 @@ globals:
         text: '[Postagem mais recente do LOOT](%1%).'
     subs: [ 'https://loot.github.io/latest-thread/' ]
 
-groups: []
+groups:
+- name: &dlcGroup DLC
+
+- name: &unofficialPatches
+  after:
+  - *dlcGroup
+
+- name: &otherMods Other Mods
+  after: 
+  - *unofficialPatches
+
+- name: &dynamicPatchGroup Dynamic Patches
+  after:
+  - *otherMods
```

Comment:
Replace `*otherMods` with `default`.

Reply:
Ok, those all seems like good changes. How do I...agree to them? "Resolve conversation"?